src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/ErrorCodes.kt

```diff
@@ -40,6 +40,16 @@ object EmailTooLong : AuthenticationRequestErrorCodes("EMAIL_TOO_LONG", "The ema
 object EmailInvalid : AuthenticationRequestErrorCodes("INVALID_EMAIL", "Enter an email address in the correct format")
 object EmailInvalidCjsm : AuthenticationRequestErrorCodes("INVALID_CJSM_EMAIL", "Enter an email address which ends with 'cjsm.net'")
 
+@Schema(oneOf = [OneTimeCodeSessionNotFound::class, OneTimeCodeTooManyAttempts::class])
+sealed class OneTimeCodeErrorCode(
+  @Schema(allowableValues = ["OTC_SESSION_NOT_FOUND", "OTC_TOO_MANY_ATTEMPTS", "OTC_NOT_FOUND"])
+  code: String,
+  userMessage: String
+) : StandardErrorCodes(code, userMessage)
+object OneTimeCodeSessionNotFound : OneTimeCodeErrorCode("OTC_SESSION_NOT_FOUND", "There is no one time code saved for the session")
```

Comment:
I'm sure I'll find it as I carry on down this PR, but I'm wondering what the difference between `OneTimeCodeSessionNotFound` and `OneTimeCodeNotFound` is .... 🤔